addons/arsenal/script_component.hpp

```diff
@@ -0,0 +1,18 @@
+#define COMPONENT arsenal
+#define COMPONENT_BEAUTIFIED Arsenal
+#include "\z\ace\addons\main\script_mod.hpp"
+
+#define DEBUG_MODE_FULL
```

Comment:
I guess clean this up and I think we can merge

Reply:
perfect